pkgs/development/python-modules/ckzg/default.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  fetchFromGitHub,
+  clang,
+  buildPythonPackage,
+  python,
+  pythonOlder,
+  pyyaml,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "ckzg";
+  version = "2.0.1";
```

```suggestion
  version = "2.0.1";
  pyproject = true;
```

All python packages should be built using a PEP517 builder, and that enables that. Requires setting up `build-system`, which you already did.

---

done